backup-stores/azure/src/main/java/io/camunda/zeebe/backup/azure/ManifestManager.java

```diff
@@ -198,6 +205,15 @@ Manifest getManifest(final BackupIdentifier id) {
     }
   }
 
+  public Collection<Manifest> listManifests(final BackupIdentifierWildcard wildcard) {
+
+    return blobContainerClient.listBlobs().stream()
```

Comment:
:x: Please correct be if I'm wrong but this listing will find not just manifests but all backup contents too. As soon as you have a couple of backups, this is very inefficient. The GCS and S3 stores solve this by creating a dedicated folder for manifests and contents, I think we should do the same here.